### DISCUSSION TITLE
Distinguishing between automatic steps and substeps in Cypress tests without using Cucumber

### DISCUSSION THREAD

**Discussion by tomaszwojciechowski** [allure-framework/allure2#2705] [Documentation] [answered]
Hello
Hello I see example under https://allurereport.org/images/js/cypress/example.png and am thinking how can I get same result in test without using Cucumber in current beta? For now I see only first command listed as substep, and rest as steps.

**Comment by baev** [comment, marked as answer]
The issue should be fixed in the `allure-cypress@3.0.8`
